migrations/versions/2e0d2e02a721_.py

```diff
@@ -0,0 +1,70 @@
+"""empty message
```

Comment:
Please add a change description.

Reply:
I think it would be best if we discuss the changes in this file in a sync meeting - I don't see the details in the PR description.